src/lib/functions/server.ts

```diff
@@ -399,15 +406,12 @@ const startWebServer = async ({
 }: {
   debug: boolean
   server: ReturnType<Awaited<typeof getFunctionsServer>>
-  settings: $TSFixMe
+  settings: FunctionsSettings
 }) => {
   await new Promise<void>((resolve) => {
-    // @ts-expect-error TS(7006) FIXME: Parameter 'err' implicitly has an 'any' type.
-    server.listen(settings.functionsPort, (err) => {
-      if (err) {
-        errorExit(`${NETLIFYDEVERR} Unable to start functions server: ${err}`)
-      } else if (debug) {
-        log(`${NETLIFYDEVLOG} Functions server is listening on ${settings.functionsPort}`)
+    server.listen(settings.functionsPort, () => {
```

Comment:
Feel free to triple-check this, but there didn't seem to be such a thing as an `err` param here

Reply:
Where do you see the err argument? I've followed the docs as far as they go, and there's no indication of such a thing. It seems to me the callback gets registered on the `"listening"` event, and errors can only be emitted viathe `'error'` event, which needs to be handled separately.

Wow, this is a confusing, under-documented space: https://github.com/nodejs/node/issues/21482

Reply:
(We touched base elsewhere. The docs are bad but all signs point toward there not being an `error` arg here.)